Add a "large message radius" toggle

rafitamolin opened the pull request:
Closes #124.

64Gram's actual behavior is:

`bubbleRadiusSmall: 6px;`
`bubbleRadiusLarge: 6px;`

Taking in consideration 64Gram uses `bubbleRadiusLarge` as default, I kept that to `6px`, which is what you prefer as the developer. However, I changed `bubbleRadiusSmall` to `16px`, because if you manually go to Settings > Advanced > Experimental Settings and turn on the toggle, 64Gram uses `bubbleRadiusSmall`, so people who like rounded bubble radius can be happy too.

This is a solution designed and carried out quickly. I know that from a development point of view it is a disaster to change it just like that, but taking into account that the developer of 64Gram makes his client as he likes and that he will not want to waste his time for free in features he does not want, I have contributed this grain of sand.

KurimuzonAkuma reviewed (commented):
I don't think this is required, since the official version has the same description.
![image](https://user-images.githubusercontent.com/31959970/206363795-2cef52c3-ca38-4af0-b6d1-ff2ff2ed8fe6.png)

rafitamolin commented:
> I don't think this is required, since the official version has the same description. ![image](https://user-images.githubusercontent.com/31959970/206363795-2cef52c3-ca38-4af0-b6d1-ff2ff2ed8fe6.png)

@KurimuzonAkuma Telegram Desktop switched from small message bubble radius to rounded ones. This was a forced change, and there was no way for users that didn't like the rounded corners to switch back. 64Gram was quickly updated and changed the bubble radius to make them small again. After a while, Telegram Desktop was updated and introduced that Experimental Settings toggle that allowed users to switch to small bubble radius.

The problem now is that 64Gram forces small bubble radius and there is no way to use rounded ones, whereas in official Telegram Desktop with that Experimental Toggle you could switch from rounded to small or viceversa when you want.

This PR fixes that problem in 64Gram, making the experimental toggle change from small bubble radius to rounded, to please everyone.
